.githooks/pre-commit

```diff
@@ -4,7 +4,7 @@ exec 1>&2
 
 # paths to check and re-format
 sources="src unittests"
-formatter="clang-format -i"
+formatter="clang-format-11 -i"
```

Comment:
Do we need to update the document to instruct the required version of clang-format?
Is "clang-format-11" a custom cmd?